src/main/java/com/cleanroommc/groovyscript/compat/mods/aether/Enchanter.java

```diff
@@ -0,0 +1,95 @@
+package com.cleanroommc.groovyscript.compat.mods.aether;
+
+import com.cleanroommc.groovyscript.api.GroovyLog;
+import com.cleanroommc.groovyscript.api.documentation.annotations.*;
+import com.cleanroommc.groovyscript.helper.Alias;
+import com.cleanroommc.groovyscript.helper.recipe.AbstractRecipeBuilder;
+import com.cleanroommc.groovyscript.registry.ForgeRegistryWrapper;
+import com.cleanroommc.groovyscript.registry.ReloadableRegistryManager;
+import com.gildedgames.the_aether.api.enchantments.AetherEnchantment;
+import net.minecraft.item.ItemStack;
+import net.minecraftforge.fml.common.registry.GameRegistry;
+import org.jetbrains.annotations.Nullable;
+
+@RegistryDescription
```

Comment:
could you add a section to the [lang file](https://github.com/CleanroomMC/GroovyScript/tree/master/src/main/resources/assets/groovyscript/lang/en_us.lang) to give the entries used by the wiki docs localization (eg `groovyscript.wiki.aether_legacy.aether_enchantment.title=Enchanter`).
once this is done, you could run `/gs generateWiki` and open a PR over at https://github.com/CleanroomMC/GroovyScript-Docs.